mediapipe/tasks/python/test/vision/image_classifier_test.py

```diff
@@ -0,0 +1,481 @@
+# Copyright 2022 The MediaPipe Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for image classifier."""
+
+import enum
+from unittest import mock
+
+import numpy as np
+from absl.testing import absltest
+from absl.testing import parameterized
+
+from mediapipe.python._framework_bindings import image as image_module
```

Comment:
@jiuqiant Should we move image to another namespace, i.e. mediapipe.framework.image? It's probably easier to make it an alias of mediapipe.python._framework_bindings.image.

Reply:
You mean reexporting mediapipe.python._framework_bindings.image as mediapipe.framework.image?

Reply:
Correct. Having `_framework_bindings` in the import path is wired ...

This can be done by simply adding the file, mediapipe/python/framework/__init__.py, and put the following code inside:
```
from mediapipe.python._framework_bindings import image as image_module

image = image_module
```